DiagnosticReport/DiagnosticReport-PhoebeSmithGeneticReport-Example.json

```diff
@@ -0,0 +1,69 @@
+{
+  "resourceType": "DiagnosticReport",
+  "id": "DiagnosticReport-PhoebeSmithGeneticReport-Example",
+  "identifier": [
+    {
+      "system": "https://mft.nhs.uk//nwglh/lims",
+      "value": "SLC52A2_GenomicReport",
+      "assigner": {
+        "identifier": {
+          "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+          "value": "R0A"
+        }
+      }
+    }
+  ],
+  "basedOn": [
+    {
+      "reference": "ServiceRequest-WGSTestOrderForm-TrioTestingProband-Example"
+    }
+  ],
+  "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/v2-0074",
+          "code": "GE",
+          "display": "Genetics"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "1054161000000101",
+        "display": "Genetic report"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-PheobeSmitham-Example"
+  },
+  "issued": "2023-10-16T15:50:00Z",
+  "performer": [
+    {
+      "identifier": {
+        "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+        "value": "R0A"
+      }
+    }
+  ],
+  "specimen": [
+    {
+      "reference": "Specimen/Specimen-BloodEDTA-WithCollectoinDetails-Example"
```

Comment:
Add specimen references for the maternal/paternal specimens when those have been added